client/apps/app-toolbox/src/app/components/table-editor/table/table.component.html

```diff
@@ -48,7 +45,7 @@
                     aria-label="Map a Class" (click)="openMappingDialog(col.colLabel, col.pk_column, col.mapping)">
                     <mat-icon svgIcon="link"></mat-icon>
                   </button>
-                  <mat-icon *ngIf="(readmode$|async) && i != 0"></mat-icon>
+                  <mat-icon *ngIf="(readmode$|async) && i != 0 && origin !== 'importer'"></mat-icon>
```

Comment:
The reason for these empty icons are to have placeholders in case of sorting, mappings, ...

Reply:
And why is your change (`&& origin !== 'importer'?`) needed?
was it not properly aligned before?
It would be great to have a new story in the table component stories, where we see the table in importer mode. This would make it easier to see the impact of your change.

Reply:
Indeed, it was not properly aligned, even now on production.
I added a Story for the importer mode. 
I checked if (on the story) this changes solves the problem. And it does.